testsuite/shared/src/main/java/org/jboss/as/test/integration/common/jms/ActiveMQProviderJMSOperations.java

```diff
@@ -233,4 +234,54 @@ public void removeSystemProperties() {
             throw new RuntimeException(e);
         }
     }
+
+    @Override
+    public void addJmsConnector(String connectorName, String socketBinding, String endpoint) {
```

Comment:
this should be renamed to addHttpConnector()  to be consistent with the management model